build-config/conf/busybox.config

```diff
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
-# Busybox version: 1.20.2
-# Wed Oct 29 14:52:19 2014
+# Busybox version: 1.25.1
+# Wed Oct 12 03:08:41 2016
```

Comment:
This new busybox config turns on a lot of features that were previously disabled (or did not exist).

Generally growing the size of ONIE is not a problem, except for some of the older powerpc platforms.

In particular does the accton/accton_as5610_52x machine build OK with this configuration?

I saw you added a mechanism for a machine to override the busyconf, which is how the older machines could be supported.